modules/ROOT/partials/configuration/autocorrect_capitalize.adoc

```diff
@@ -0,0 +1,23 @@
+[[autocorrect_capitalize]]
+
+== `+autocorrect_capitalize+`
+
+This option enables or disables the autocorrection for capitalization errors when the editor is loaded. When set to `false`, the autocorrection will be disabled when the editor is initialized.
+
+NOTE: The user's preference takes precendence over the configuration at the next initialization. The user's preference is saved to `localStorage` when the user enables or disables the autocorrection via menu item.
```

Comment:
Why do we have the spelling note in autocorrect but not here?

Also this note is duplicated word for word and should be extracted out into a partial to remove the duplication across the options.

Reply:
Oh right. Forgot that capitalization is also disabled when the spelling language is not English